pkg/controller/volume/attachdetach/attach_detach_controller.go

```diff
@@ -818,6 +819,11 @@ func (adc *attachDetachController) GetSubpather() subpath.Interface {
 	return nil
 }
 
+func (adc *attachDetachController) GetFilteredDialOptions() *proxyutil.FilteredDialOptions {
+	// FilteredDial not needed in attachdetach controller
```

Comment:
If the targeted plugins implement Attacher, then we do need it.

Reply:
Done.